packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx

```diff
@@ -0,0 +1,144 @@
+import { useTranslation } from 'react-i18next';
+import {
+  Box,
+  Button,
+  Checkbox,
+  FormControl,
+  InputLabel,
+  ListItemText,
+  MenuItem,
+  OutlinedInput,
+  Select,
+  SelectChangeEvent,
+  Stack,
+  Typography,
+} from '@mui/material';
+import { CustomDropdown } from '@pagopa-pn/pn-commons';
+import SearchIcon from '@mui/icons-material/Search';
+import { useState } from 'react';
+
+const ITEM_HEIGHT = 48;
```

Comment:
Remove the mock data and the custom style

Reply:
done